css/styles.css

```diff
@@ -175,10 +175,10 @@ h4 {
 .copyright {
   text-align: left;
   position: absolute; 
-  font-size: 0.7vw; 
+  font-size: 0.7rem;
   color: #666; 
   z-index: 1;
-  padding-top: 79vh; 
+  bottom: 0;
```

Comment:
Instead of pushing the copyright down from the top, you could just position it from the bottom of its container (the globe element).